web/sites/default/config/core.entity_view_display.node.news.featured.yml

```diff
@@ -83,13 +83,13 @@ content:
             fis-def-at: false
             fi-def-at: false
   field_published_on:
-    type: datetime_default
```

Comment:
Why did you switch this from a date format that we already have? The more correct way to address this would be to update the Month Day Year format.

Reply:
It's so that the pattern can actually receive a meaningful prop (e.g. string to be formatted by the pattern and inject within our own `<time>` tag inside the pattern) rather than an already rendered chunk of markup (in this case, `<time datetime="March 26, 2020">March 26, 2020</time>` which would then get passed into our `<time>` tag and break everything). When working with design system patterns its pretty standard practice to strip away any tags or rendered markup unless very necessary, then inject your raw data into your pattern where needed.

Reply:
I'm not sure I agree with this. Instead of having one clear place to go to set time formatting, it now has to be done on every single content config that has date information. That makes it really hard to track down a bug and adds lots of frivolous lines of code. (Not that Drupal is terribly DRY, but this makes it a lot worse.)

I would rather the pattern take the rendered Drupal fields. That way we only override where we need to. Remember, we are not implementing a design system across a lot of sites, we just need a way to theme easier. Let's chat a bit more about this on Zoom tomorrow.